pennylane_lightning/src/tests/Makefile

```diff
@@ -24,15 +24,24 @@ all: test
 lightning_unittest.o: lightning_unittest.cpp
 	$(CC) $^ $(CFLAGS) -c
 
+lightning_gates_unittest.o: lightning_gates_unittest.cpp
+	$(CC) $^ $(CFLAGS) -c
+
 gtest_main.o: gtest_main.cpp
 	$(CC) $^ $(CFLAGS) -c
 
 cpptests: gtest_main.o lightning_unittest.o
 	$(CC) $^ -o $@ $(LFLAGS)
 
+cpptests_new: gtest_main.o lightning_gates_unittest.o
+	$(CC) $^ -o $@ $(LFLAGS)
+
 test: cpptests
 	./cpptests
 
+test_new: cpptests_new
+	./cpptests_new
+
```

Comment:
Do we want to separate this out? Currently, I do `make test-cpp` from the top-level directory, which I believe wouldn't run these tests if we make a `test_new` and `cpptests_new`. In other words, why don't we put everything in `cpptests`, which is accessible by the main Makefile? (I'm also conscious that eventually the new backend will no longer be new, and not wanting to delete lots of `_new` everywhere)

Reply:
changed the makefile so `lightning_gates_unittest` is now run with `make test`